### DISCUSSION TITLE
Scene icons not changing state colour

### DISCUSSION THREAD

**Discussion by jayjayvm** [Blackymas/NSPanel_HA_Blueprint#690] [Q&A]
Title summs it up already, when i enable a scene the icon of the scene will not get lit on the panel, howto solve that or ma I the only one experiencing this?

**Comment by deejaybeam** [comment]
A pic would be helpful, but:
- there is/was (depends of the installed version) a problem regarding the icons of scenes and buttons -> assignend to the buttons
- the background can´t be changed because a scene work like a button - if it´s pressed, it is called, but has no state on/off

**Comment by jayjayvm** [comment]
Yes true, I had hoped you had found a work around, sure I did number my scenes too with helpers, that way you can scroll easier through them with a pushbutton. But those can not linked and used :)
But I see the problem, thank you

**Comment by edwardtfn** [comment]
It should have some feedback to the user when a button representing a scene is pressed... Like showing on for a second and then off again... Same for buttons, etc. 
I'm not sure if it is like this currently l but I think so... Could you please confirm?

**Comment by edwardtfn** [comment]
Fixed in DEV (#725) and will be included in a coming release.

More of this same discussion in #691.
